src/app/views/home/home.component.html

```diff
@@ -15,25 +15,73 @@
       </div>
     </div>
   </div>
-  <div class="row home-new-jobs">
-    <div class="col-xs-12 col-sm-9 title">
-      <h2>{{'start.job_section.title' | translate}}</h2>
+
+
+  <div class="row home-slider-container my-jobs">
+    <div class="col-xs-12 my-jobs-title">
+      <h2>{{'home_page.new_jobs_slider.title' | translate}}</h2>
     </div>
-    <div class="col-xs-12 col-sm-3 show-all">
-      <span routerLink="/jobs/1">{{'common.show_all' | translate | lowercase}} <i class="icon-ic-arrow-right"></i></span>
+    <div class="col-xs-12 my-jobs-show-all-button">
+      <span routerLink="/jobs/1">{{'common.show_all' | translate | lowercase}} > <i class="icon-ic-arrow-right"></i></span>
     </div>
-    <div class="col-xs-12 jobs-slider">
+    <div class="col-xs-12 jobs-slider" align="center">
       <div *ngFor="let job of newJobs" class="item-container clickable" [routerLink]="['/job', job.id]">
-        <div class="job-name"><div>
-          <translation-item [text]="job.name" [translation]="job.translated.name" [short]="true" [length]="35"></translation-item>
-        </div></div>
-        <div class="salary-container">
+        <div class="job-name">
+          <div>
```

Comment:
Add class. Convention: All html elements should have a class that clearly states just by looking at the class where this class is used in the html.